rir/src/api.cpp

```diff
@@ -133,6 +133,10 @@ REXPORT SEXP pir_compile(SEXP what, uint32_t verbose, SEXP dryRun_) {
     return what;
 }
 
+REXPORT SEXP pir_compile(SEXP what, SEXP verbose, SEXP dryRun_) {
+    return pir_compile_(what, (uint32_t)INTEGER(INTEGER(verbose))[0], dryRun_);
```

Comment:
Why twice `INTEGER`?

Also, this will segv, when called with anything but an INTSXP, so it needs checks, that gracefully handle non integer values (like throw an R level error...). Something like
`if (TYPEOF(verbose) != INTSXP || Rf_length(verbose) < 1) error...`